.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync to Upstream
```

Comment:
This is to pull changes from this repository into forks to keep them up to date, It will have no effect on the original repository

Reply:
Hi @cleder ,
Could you please provide more info about that one.
I did get the main idea of that workflow.